Sources/Purchasing/Purchases.swift

```diff
@@ -1632,194 +1637,49 @@ public extension Purchases {
-            observerMode: observerMode,
-            userDefaults: userDefaults,
-            useStoreKit2IfAvailable: false
-        )
-    }
-
-    /**
-     * Configures an instance of the Purchases SDK with a custom userDefaults.
+     * Configures an instance of the Purchases SDK with a specified API key.
      *
-     * Use this constructor if you want to sync status across a shared container,
-     * such as between a host app and an extension.
-     * The instance of the `Purchases` SDK will be set as a singleton.
+     * The instance will be set as a singleton.
      * You should access the singleton instance using ``Purchases/shared``
      *
-     * - Parameter apiKey: The API Key generated for your app from https://app.revenuecat.com/
-     *
-     * - Parameter appUserID: The unique app user id for this user. This user id will allow users to share their
-     * purchases and subscriptions across devices. Pass `nil` or an empty string if you want ``Purchases``
-     * to generate this for you.
-     *
-     * - Parameter observerMode: Set this to `true` if you have your own IAP implementation and want to use only
-     * RevenueCat's backend. Default is `false`.
-     *
-     * - Parameter userDefaults: Custom `UserDefaults` to use
-     *
-     * - Parameter useStoreKit2IfAvailable: EXPERIMENTAL. opt in to using StoreKit 2 on devices that support it.
-     * Purchases will be made using StoreKit 2 under the hood automatically.
-     * - Important: Support for purchases using StoreKit 2 is currently in an experimental phase.
-     * We recommend setting this value to `false` (default) for production apps.
-     *
-     * - Returns: An instantiated ``Purchases`` object that has been set as a singleton.
-     */
-    @objc(configureWithAPIKey:appUserID:observerMode:userDefaults:useStoreKit2IfAvailable:)
-    @discardableResult static func configure(withAPIKey apiKey: String,
-                                             appUserID: String?,
-                                             observerMode: Bool,
-                                             userDefaults: UserDefaults?,
-                                             useStoreKit2IfAvailable: Bool) -> Purchases {
-        configure(
-            withAPIKey: apiKey,
-            appUserID: appUserID,
-            observerMode: observerMode,
-            userDefaults: userDefaults,
-            useStoreKit2IfAvailable: useStoreKit2IfAvailable,
-            dangerousSettings: nil
-        )
-    }
-
-    /**
-     * Configures an instance of the Purchases SDK with a custom userDefaults.
-     *
-     * Use this constructor if you want to sync status across a shared container,
-     * such as between a host app and an extension.
-     * The instance of the `Purchases` SDK will be set as a singleton.
-     * You should access the singleton instance using ``Purchases/shared``
+     * - Note: Use this initializer if your app does not have an account system.
```

Comment:
Not relevant either? Depends on how the `Configuration` is defined?

Reply:
I kinda want to keep this one around since I think it's the most common use case we have.

Reply:
I'm confused, this is the only method, so you'd use it whether you have an account system or not?

Reply:
We have both:
`@discardableResult static func configure(with configuration: Configuration) -> Purchases {`

and 
`@discardableResult static func configure(withAPIKey apiKey: String) -> Purchases {`

The first is the preferred method for folks who want to do more than just set an API key.
The second is for people how have the most simple setup and don't require customization. Like if they don't have an account system you wouldn't set the `appUserID` during init. But if you did, you could use the `Configuration.Builder` to set that property.

Reply:
Oooh sorry, that makes sense 👍🏻

Reply:
🎉